recipes/cpp-member-accessor/all/conanfile.py

```diff
@@ -0,0 +1,43 @@
+import os
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import get, copy
+from conan.tools.layout import basic_layout
+
+class CppMemberAccessorConan(ConanFile):
+    name = "cpp-member-accessor"
+    version = "1.0.0"
+    license = "MIT"
+    author = "Hubert Liberacki <hliberacki@gmail.com>"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/hliberacki/cpp-member-accessor"
+    topics = ("c++", "templates", "access-private-members", "header-only")
+    settings = "os", "arch", "compiler", "build_type"
+    exports_sources = "include/*"
+    no_copy_source = True
+    package_type = "header-library"
+    exports = "LICENSE"
+
+    @property
+    def _min_cppstd(self):
+        return 14
+
+    def validate(self):
+        check_min_cppstd(self, self._min_cppstd)
```

Comment:
```suggestion
   def package_id(self):
        self.info.clear()

    def validate(self):
        check_min_cppstd(self, 14)
```

Need to clear settings from the package ID, otherwise, the package will follow settings too, but it content will be always the same.